t/lib/Helper.pm

```diff
@@ -6,6 +6,12 @@ use Test::Fatal ();
 use Path::Tiny ();
 use File::pushd ();
 use Capture::Tiny ();
+use Config;
+
+# Microsoft nmake outputs a copyright message that
+# messes up the output checks, but we can work around
+# this by setting the -nologo option using MAKEFLAGS
+$ENV{MAKEFLAGS} = 'nologo' if $Config{make} eq 'nmake';
```

Comment:
This should instead be: `$ENV{MAKEFLAGS} = join(':', 'nologo', ($ENV{MAKEFLAGS} // ()));` so as to not wipe out any existing settings.

I'm looking into whether this can/should go into Dist::Zilla::Plugin::MakeMaker instead.

Reply:
rjbs rightly pointed out that : might not be the right delimiter here though, doh!

Reply:
If there is a separator character for MAKEFLAGS for nmake it isn't `:` (or `;`):

```
C:\Users\ollisg>set MAKEFLAGS=nologo:foo

C:\Users\ollisg>nmake
NMAKE : fatal error U1065: invalid option ':'
Stop.
```

Reply:
updated with f480d7a using a space as the separator which by experimentation with nmake directly seems to be the right thing.  I haven't found any documentation to confirm this, but the tests do pass with or without `MAKEFLAGS` set.